tests/Tests/600__ai_apps/620__intel_aikit/620__intel_aikit.robot

```diff
@@ -25,7 +25,7 @@ Verify intel aikit Is Available In RHODS Dashboard Explore Page
   Verify Service Provides "Get Started" Button In The Explore Page     ${intel_aikit_container_name}
 
 Verify Inetl AIKIT Operator Can Be Installed Using OpenShift Console
-   [Tags]  ODS-760   ODS-702   Sanity
+   [Tags]  ODS-760   ODS-702   Tier2
```

Comment:
We should try to keep tags formatting consistent, could you update it to
```
[Tags]  Tier2
...    ODS-XYZ
```
?

Reply:
done